tests/Microsoft.Identity.Test.Unit/WwwAuthenticateParametersTests.cs

```diff
@@ -178,6 +178,28 @@ public async Task CreateFromResourceResponseAsync_HttpClientFactory_Async()
             httpClientFactory.Received().GetHttpClient();
         }
 
+        [TestMethod]

```

Comment:
Is there a unit test that checks that if the WWWAuthenticate header has an ADFS or B2C authority, the tenant id is null?

Reply:
Will add

Reply:
Please assist me in authoring the test when it uses `TestConstants.B2CAuthority` as the tenant id doesn't come `null`.

Reply:
Ah, sorry about @abatishchev , I made a mistake. B2C is tenanted, https://login.microsoftonline.in/tfp/tenant/{B2CSignUpSignIn}/ has tenantId `tenant`.

Reply:
@bgavrilMS cool, thanks. updated the tests. please take a look again.